docs/user_manual.md

```diff
@@ -878,6 +879,157 @@ datum stats -p test_project
 
 </details>
 
+
+### Validate project annotations
+
+This command inspects annotations based on the task type
+and stores the result in JSON file.
+
+The task types supported are `classification`, `detection`, and `segmentation`.
+
+The validation result contains
+- annotation statistics based on the task type
+- validation reports, such as
+    - items not having annotations
+    - items having undefined annotations
+    - imbalanced distribution in class/attributes
+    - too small or large values (items_far_from_mean)
```

Comment:
`items_far_from_mean` - it's not clear, what it means here.

Reply:
`items_far_from_mean` is a keyword of `numerical_stat_template`.
It means that a value in distribution is too big or too small to the avaerage value(=mean) of the distribution.
I think it's better to remove `(items_far_from_mean)`.